zone/worldserver.cpp

```diff
@@ -3184,6 +3184,11 @@ void WorldServer::HandleMessage(uint16 opcode, const EQ::Net::Packet &p)
 			LogError("Loading npcs faction lists failed!");
 		}
 
+		LogInfo("Loading splash faction lists");
+		if (!database.LoadSplashFaction(hotfix_name)) {
```

Comment:
Possibly `content_db` like the others

Reply:
I'm guessing for shared memory the database doesn't matter outside of (re)loading from the real database since it's just local, just some of the inconsistencies are confusing